certified-connectors/Tribal/apiProperties.json

```diff
@@ -0,0 +1,297 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "EventsConnectorClient",
```

Comment:
@mszy-tribal Can you please replace it with `"[[TEST]]"` or `"[[DUMMY]]"`.

Reply:
This has been replaced